bin/autojump

```diff
@@ -135,10 +135,13 @@ def add_path(data, path, weight=10):
     with resulting duplicate entries in the database than a single canonical
     path.
     """
-    path = unico(path).rstrip(os.sep)
```

Comment:
Ah, I see. I guess there is no reason for this line not to be here, adding it back in before passing path to abspath shouldn't cause a problem.